test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go

```diff
@@ -31,7 +31,8 @@ const (
 	cyclictestMaxLatency = "CYCLICTEST_MAXIMUM_LATENCY"
 	latencyTestCpus      = "LATENCY_TEST_CPUS"
 	//invalid values error messages
-	unexpectedError = "Unexpected error"
+	unexpectedError      = "Unexpected error"
+	contextDeadlineError = "context deadline exceeded"
```

Comment:
if there is no specific error to latency configuration let's use the general one `unexpectedError`

Reply:
Okay, Resolved